diff -Naur Scotch-orig/src/esmumps/Makefile Scotch/src/esmumps/Makefile
--- Scotch-orig/src/esmumps/Makefile	2013-06-21 16:11:13.488345358 +0200
+++ Scotch/src/esmumps/Makefile	2013-06-21 16:10:25.625970173 +0200
@@ -44,7 +44,7 @@
 				$(CC) $(CFLAGS) $(CLIBFLAGS) -I$(includedir) -c $(<) -o $(@)
 
 %$(EXE)	:	%.c
-		 		$(CC) $(CFLAGS) -I$(includedir) $(<) -o $(@) -L$(libdir) $(LDFLAGS) -L. -l$(ESMUMPSLIB) -l$(SCOTCHLIB) -l$(SCOTCHLIB)errexit
+		 		$(CC) $(CFLAGS) -I$(includedir) $(<) -o $(@) -L$(libdir) $(LDFLAGS) -L. -l$(ESMUMPSLIB) -l$(SCOTCHLIB) -l$(SCOTCHLIB)errexit -lm
 
 ##
 ##  Project rules.
